source/_docs/guides/drupal-8-commerce.md

```diff
@@ -0,0 +1,122 @@
+   composer require "drupalcommerce/commerce_base dev-8.x-1.x" "drupal/commerce:~2.0"  "drupal/admin_toolbar:~1.0"  "drupal/swiftmailer:~1.0"
+   ```
+
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p markdown="1">The commands above include dependencies required but not listed for `commerce_base`.  simpler once [a few Composer changes are made within the Commerce Installation Profile itself](https://github.com/drupalcommerce/commerce_base/pull/7){.external}.</p>
+    </div>
+
+3. Running `git status` should show that the `composer.json` and `composer.lock` files have changed:
+
+   ![Git Status showing updated Composer files](/source/docs/assets/images/guides/drupal-8-commerce/git-status.png)
+
+4. Commit the new files and push them to GitHub:
+
+   ```bash
+   git commit -am "add commerce_base to project"
+   git push origin master
+   ```
+
+5. From your [CircleCI Dashboard](https://circleci.com/dashboard){.external} you can see that tests are already being run on your new commit. Once they pass, CircleCI will push the changes to your Site Dashboard.
+
+6. Go to your newly created Site Dashboard. Under the <span class="glyphicons glyphicons-wrench"></span> **Dev** tab, click on <span class="glyphicons glyphicons-embed-close"></span> **Code**, then **install later**. You should now see your commit history. Once CircleCI completes the automated tests built into our repository, it will commit the build assets and push them to Dev:
+
+    ![Build Assets on Dev](/source/docs/assets/images/guides/drupal-8-commerce/build-assets.png)
+
+## Reinstall Drupal
+
+1. The Build Tools Plugin command we used earlier automatically installed Drupal's standard profile in the Dev environment for us. Now that we've installed the Commerce profile, we want that installed instead. Using Terminus, we can run the Drush command `site-install` which will first clear the database of the Standard profile before installing Commerce. This Drush command requires that the system be in writable (SFTP) mode:
+
+   ```bash
+   terminus connection:set $SITENAME.dev sftp
+   terminus drush $SITENAME.dev -- site-install commerce
+   ```
+
+   Review the last two lines of output to identify the username and password created:
+
+   ```bash
+   Installation complete.  User name: admin  User password: jTHD8hd85U         [ok]
+   Congratulations, you installed Drupal!                                  [status]
+   ```
+
+
+2. Log in to your Drupal site in the Dev environment. The presence of the **Commerce** button on the toolbar indicates a succefull install:
+
+    ![Drupal Commerce in the Toolbar](/source/docs/assets/images/guides/drupal-8-commerce/commerce-button.png)
+
+## Conclusion
+
+What you do next is up to you and your needs. Remember that you're now using Composer to manage modules and dependencies for your site. Consider reading our [Composer Fundamentals and Workflows](/docs/composer) doc for more information.
```

Comment:
Is it worth including that core is also managed via Composer (or should be)? https://pantheon.io/docs/composer/#managing-core-as-a-project-dependency